compiler/src/iree/compiler/Codegen/Common/VectorLayoutAnalysis.cpp

```diff
@@ -224,7 +224,7 @@ ChangeResult DistributionLayout::resolveWithPossibleConflict(
   resolvedLayout->subscribeEnforcement(enforcement);
 
   // We can now resolve this resolved value to the required layout.
-  resolvedLayout->resolve(rhs);
+  (void)resolvedLayout->resolve(rhs);
```

Comment:
Can this fail? Should we add an assertion?

Reply:
resolve can never fail. It just tells you if the layout was changed or not. Here, we know already that it will change, so we ignore it.